libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/PredictiveCacheController.kt

```diff
@@ -332,8 +345,14 @@ class PredictiveCacheController @JvmOverloads constructor(
         } else {
             val descriptorOptions = TilesetDescriptorOptions.Builder()
                 .styleURI(styleURI)
-                .minZoom(MIN_ZOOM)
-                .maxZoom(MAX_ZOOM)
+                .minZoom(
+                    predictiveCacheOptions.predictiveCacheMapsOptions
+                        .predictiveCacheZoomLevelOptions.minZoom
+                )
+                .maxZoom(
+                    predictiveCacheOptions.predictiveCacheMapsOptions
+                        .predictiveCacheZoomLevelOptions.minZoom
```

Comment:
maxZoom
There should be unit tests that would fail...

Reply:
great catch 👍